src/docfx/build/page/AttributeTransformer.cs

```diff
@@ -32,13 +32,15 @@ object TransformContent(IEnumerable<DataTypeAttribute> attributes, object value,
                     var (html, markup) = Markup.ToHtml((string)value, file, ReadFileDelegate, GetLinkDelegate, ResolveXrefDelegate, null, MarkdownPipelineType.Markdown);
                     errors.AddRange(markup.Errors);
                     result = html;
+                    XrefMap.HandleDependencyMap(markup, file, callback?.DependencyMapBuilder);
```

Comment:
>callback [](start = 62, length = 8)

this callback is really confused for me

Reply:
I can expand it

Reply:
no hurry, let's together find a easy and simple way

---
In reply to: [235588157](https://github.com/dotnet/docfx/pull/3646#discussion_r235588157) [](ancestors = 235588157)